ClimsoftVer4/ClimsoftVer4/formAWSRealTime.vb

```diff
@@ -1,4 +1,20 @@
-﻿Imports System.IO
+﻿'CLIMSOFT - Climate Database Management System
+' Copyright (C) 2017
+'
+' This program is free software: you can redistribute it and/or modify
+' it under the terms of the GNU General Public License as published by
+' the Free Software Foundation, either version 3 of the License, or
+' (at your option) any later version.
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
... after the licence information, we would usually decide whether to have no black lines or one blank line.
Either is fine - but we should be consistent and do the same in every file.